types/src/error.rs

```diff
@@ -80,6 +80,36 @@ impl<'a> fmt::Display for ErrorResponse<'a> {
 	}
 }
 
+/// The return type of the subscription's method for the rpc server implementation.
+///
+/// **Note**: The error does not contain any data and is discarded on drop.
+pub type ReturnTypeSubscription = Result<(), SubscriptionEmptyError>;
```

Comment:
```suggestion
pub type SubscriptionResult = Result<(), SubscriptionEmptyError>;
```

Reply:
that one is likely only internal in jsonrpsee, you can rename it to `InnerSubscriptionResult` or something.

and rename :point_up_2: to `SubscriptionResult`